frontend/public/tinymce/langs/zh_TW.js

```diff
@@ -0,0 +1,419 @@
+"Previous": "\u4e0a\u4e00\u500b",
+"Next": "\u4e0b\u4e00\u500b",
+"Find and replace...": "\u5c0b\u627e\u53ca\u53d6\u4ee3...",
+"Could not find the specified string.": "\u7121\u6cd5\u67e5\u8a62\u5230\u6b64\u7279\u5b9a\u5b57\u4e32",
+"Match case": "\u76f8\u5339\u914d\u6848\u4ef6",
+"Find whole words only": "\u50c5\u627e\u51fa\u5b8c\u6574\u5b57\u532f",
+"Spell check": "\u62fc\u5beb\u6aa2\u67e5",
+"Ignore": "\u5ffd\u7565",
+"Ignore all": "\u5ffd\u7565\u6240\u6709",
+"Finish": "\u5b8c\u6210",
+"Add to Dictionary": "\u52a0\u5165\u5b57\u5178\u4e2d",
+"Insert table": "\u63d2\u5165\u8868\u683c",
+"Table properties": "\u8868\u683c\u5c6c\u6027",
+"Delete table": "\u522a\u9664\u8868\u683c",
+"Cell": "\u5132\u5b58\u683c",
+"Row": "\u5217",
+"Column": "\u884c",
+"Cell properties": "\u5132\u5b58\u683c\u5c6c\u6027",
+"Merge cells": "\u5408\u4f75\u5132\u5b58\u683c",
+"Split cell": "\u5206\u5272\u5132\u5b58\u683c",
+"Insert row before": "\u63d2\u5165\u5217\u5728...\u4e4b\u524d",
+"Insert row after": "\u63d2\u5165\u5217\u5728...\u4e4b\u5f8c",
+"Delete row": "\u522a\u9664\u5217",
+"Row properties": "\u5217\u5c6c\u6027",
+"Cut row": "\u526a\u4e0b\u5217",
+"Copy row": "\u8907\u88fd\u5217",
+"Paste row before": "\u8cbc\u4e0a\u5217\u5728...\u4e4b\u524d",
+"Paste row after": "\u8cbc\u4e0a\u5217\u5728...\u4e4b\u5f8c",
+"Insert column before": "\u63d2\u5165\u6b04\u4f4d\u5728...\u4e4b\u524d",
+"Insert column after": "\u63d2\u5165\u6b04\u4f4d\u5728...\u4e4b\u5f8c",
+"Delete column": "\u522a\u9664\u884c",
+"Cols": "\u6b04\u4f4d\u6bb5",
+"Rows": "\u5217",
+"Width": "\u5bec\u5ea6",
+"Height": "\u9ad8\u5ea6",
+"Cell spacing": "\u5132\u5b58\u683c\u5f97\u9593\u8ddd",
+"Cell padding": "\u5132\u5b58\u683c\u7684\u908a\u8ddd",
+"Show caption": "\u986f\u793a\u6a19\u984c",
+"Left": "\u5de6\u908a",
+"Center": "\u4e2d\u9593",
+"Right": "\u53f3\u908a",
+"Cell type": "\u5132\u5b58\u683c\u7684\u985e\u578b",
+"Scope": "\u7bc4\u570d",
+"Alignment": "\u5c0d\u9f4a",
+"H Align": "\u6c34\u5e73\u4f4d\u7f6e",
+"V Align": "\u5782\u76f4\u4f4d\u7f6e",
+"Top": "\u7f6e\u9802",
+"Middle": "\u7f6e\u4e2d",
+"Bottom": "\u7f6e\u5e95",
+"Header cell": "\u6a19\u982d\u5132\u5b58\u683c",
+"Row group": "\u5217\u7fa4\u7d44",
+"Column group": "\u6b04\u4f4d\u7fa4\u7d44",
+"Row type": "\u884c\u7684\u985e\u578b",
+"Header": "\u6a19\u982d",
+"Body": "\u4e3b\u9ad4",
+"Footer": "\u9801\u5c3e",
+"Border color": "\u908a\u6846\u984f\u8272",
+"Insert template...": "\u63d2\u5165\u6a23\u7248...",
+"Templates": "\u6a23\u7248",
+"Template": "\u6a23\u677f",
+"Text color": "\u6587\u5b57\u984f\u8272",
+"Background color": "\u80cc\u666f\u984f\u8272",
+"Custom...": "\u81ea\u8a02",
+"Custom color": "\u81ea\u8a02\u984f\u8272",
+"No color": "No color",
+"Remove color": "\u79fb\u9664\u984f\u8272",
+"Table of Contents": "\u76ee\u9304",
+"Show blocks": "\u986f\u793a\u5340\u584a\u8cc7\u8a0a",
+"Show invisible characters": "\u986f\u793a\u96b1\u85cf\u5b57\u5143",
+"Word count": "\u8a08\u7b97\u5b57\u6578",
+"Count": "\u8a08\u7b97",
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 2 spaces but found 0.](https://app.codacy.com/gh/dataease/dataease/pullRequest?prid=7687804)
```suggestion
  "Count": "\u8a08\u7b97",
```